modules/Makefile.kubernetes

```diff
@@ -62,15 +58,33 @@ ifeq ($(strip $(KUBECTL_SSH_USER)),)
 	KUBECTL_SSH_USER = $(shell ssh git@github.com 2>&1 | grep 'successfully authenticated' | cut -d' ' -f2 | cut -d'!' -f1)
 endif
 
+# More kubectl specific settings
+KUBECTL_SSH_TUNNEL ?= $(KUBECTL_SSH_USER)@$(CLUSTER_BASTION)
+KUBECTL_SSH_SOCK ?= /tmp/kubectl-$(KUBECTL_SSH_TUNNEL)
```

Comment:
I used this script to deploy to gladly last night so it does work in environments where you don't need MFA. Are you concerned about access to `/tmp`?